superset/assets/src/dashboard/stylesheets/filter-scope-selector.less

```diff
@@ -0,0 +1,241 @@
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+@import "../../../stylesheets/less/cosmo/variables.less";
+
+.filter-scope-container {
+  font-size: 14px;
+
+  .nav.nav-tabs {
+    border: none;
+  }
+}
+
+.filter-scope-header {
+  h4 {
+    margin-top: 0;
+  }
+
+  .selected-fields {
+    margin: 12px 0 16px;
+    visibility: hidden;
+
+    &.multi-edit-mode {
+      visibility: visible;
+    }
+
+    .selected-scopes {
+      padding-left: 5px;
+    }
+  }
+}
+
+.filters-scope-selector {
+  margin: 10px -24px 20px;
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  border: 1px solid #ccc;
+  border-left: none;
+  border-right: none;
+
+  a, a:active, a:hover {
+    color: @almost-black;
+    text-decoration: none;
+  }
+
+  .react-checkbox-tree .rct-icon.rct-icon-expand-all,
+  .react-checkbox-tree .rct-icon.rct-icon-collapse-all {
+    font-size: 13px;
+    font-family: @font-family-sans-serif;
+    color: @brand-primary;
+
+    &::before {
+      content: '';
+    }
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .filter-field-pane {
+    position: relative;
+    width: 40%;
+    padding: 16px 16px 16px 24px;
+    border-right: 1px solid #ccc;
+
+    .filter-container {
+      label {
+        font-weight: normal;
+        margin: 0 0 0 16px;
+      }
+    }
+
+    .filter-field-item {
+      height: 35px;
+      display: flex;
+      align-items: center;
+      padding: 0 24px;
+      margin-left: -24px;
+
+      &.is-selected {
+        border: 1px solid #aaa;
+        border-radius: 4px;
+        background-color: #eee;
+        margin-left: -25px;
+      }
+    }
+
+    .react-checkbox-tree {
+      .rct-text {
+        height: 40px;
+      }
+    }
+  }
+
+  .filter-scope-pane {
+    position: relative;
+    flex: 1;
+    padding: 16px 24px 16px 16px;
+  }
+
+  .react-checkbox-tree {
+    flex-direction: column;
+    color: @almost-black;
+    font-size: 14px;
+
+    .filter-scope-type {
+      padding: 8px 0;
+      display: block;
+
+      .type-indicator {
+        position: relative;
+        top: 3px;
+        margin-right: 8px;
+      }
+
+      &.chart {
+        font-weight: normal;
+      }
+
+      &.selected-filter {
+        padding-left: 28px;
+        position: relative;
+        color: #aaa;
+
+        &::before {
+          content: " ";
```

Comment:
it looks like you used single quotes for content above and double quotes here. not sure which is best for css, but can we make it consistent?